### PR TITLE
#1410: WeAreOnline

### PR DESCRIPTION
**Pull request by yegor256**
In order to get rid of static method in one of our tests, let's use "Conditional Test Execution" via JUnit5 extensions.

### CODE REVIEW

**Comment by yegor256** [comment]
@volodya-lombrozo please, take a look

File: eo-maven-plugin/src/test/java/org/eolang/maven/WeAreOnline.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+/**
+ * JUnit's extension to skip a test if we are not online.
+ *
+ * @since 0.26
+ */
+final class WeAreOnline implements ExecutionCondition {
+
+    @Override
+    public ConditionEvaluationResult evaluateExecutionCondition(
+        final ExtensionContext context) {
```
Comment (volodya-lombrozo) [inline eo-maven-plugin/src/test/java/org/eolang/maven/WeAreOnline.java:43]: I'm not sure here, but maybe it's better to move `) {` to the next line.

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
Only few comments

File: eo-maven-plugin/src/test/java/org/eolang/maven/WeAreOnline.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        ConditionEvaluationResult ret;
+        final String host = "www.objectionary.com";
+        final int delay = (int) TimeUnit.SECONDS.toMillis(1L);
```
[the remark this answers is not in the document]
Comment (yegor256) [reply eo-maven-plugin/src/test/java/org/eolang/maven/WeAreOnline.java:46]: @volodya-lombrozo right, fixed

File: eo-maven-plugin/src/test/java/org/eolang/maven/WeAreOnline.java (hunk 1 above)
Comment (yegor256) [reply eo-maven-plugin/src/test/java/org/eolang/maven/WeAreOnline.java:43]: @volodya-lombrozo I prefer this style of method params. I know you like another one, but both of them are fine for Qulice.

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**